src/decoder/image.rs

```diff
@@ -383,13 +416,54 @@ impl Image {
         }
     }
 
+    pub(crate) fn chunk_info(&self, chunk_index: usize) -> Option<ChunkInfo> {
+        match &self.chunk_type {
+            ChunkType::Strip => {
+                let rows_per_strip = self.strip_decoder.as_ref()?.rows_per_strip as usize;
```

Comment:
The `?` should be an unwrap since if chunk_type is ChunkType::Strip then strip_decoder should always be Some.